lib/models/addon.js

```diff
@@ -448,38 +448,68 @@ Addon.prototype.compileStyles = function(tree) {
 
   @private
   @method compileTemplates
-  @param {Tree} tree Templates file tree
+  @param {Tree} addonTree Addon file tree
   @return {Tree} Compiled templates tree
 */
-Addon.prototype.compileTemplates = function(tree) {
+Addon.prototype.compileTemplates = function(addonTree) {
   this._requireBuildPackages();
 
-  if (tree) {
+  var isTemplateInAddon = false;
```

Comment:
I'm sure someone can point out a more efficient way to determine if there is a template in the addons directory.